include/glow/Importer/ProtobufLoader.h

```diff
@@ -181,8 +189,62 @@ class ProtobufLoader {
   /// \returns the Placeholder for the external output with \p name.
   /// \pre outputVarsByName_.find(name) != outputVarsByName_.end()
   llvm::Expected<Placeholder *> getOutputByName(llvm::StringRef name) const;
+
+  /// \returns True if the operator with name \p typeName having input node
+  /// list as \p inputs is constant foldable.
+  bool isConstantFoldable(llvm::SmallVector<NodeValue, 4> &inputs,
+                          std::string typeName) const;
 };
 
+/// \returns success if the folding of operator \p op in the loader
+/// \p pLoader is successful. The folding utility uses temporary
+/// loader \p loaderCF, and associated temporary function \p F.
+template <class LoaderType, class OpType>
+llvm::Error constantFoldInLoader(Function *F, LoaderType &loaderCF,
+                                 LoaderType *pLoader, const OpType &op) {
+  PlaceholderBindings bindingCF;
+  std::vector<Tensor *> pTensorVec;
```

Comment:
`pTensorVec` does not sounds like a Glow name ;-) Could you use something like `xyzTensors`, e.g. `resultTensors` or may be just `results`?